src/web/mjs/connectors/Sukima.mjs

```diff
@@ -0,0 +1,191 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Sukima extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'sukima';
+        super.label = 'Sukima';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://www.sukima.me';
+        this.language = 'japanese';
+        this.requestOptions.headers.set( 'x-referer', this.url );
+        this.lock = false;
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/api/book/v1/free/', this.url), {
+            method: 'POST',
+            body: '{"store":false,"genre":"0"}',
+            headers: {
+                'Content-Type': 'application/json;charset=utf-8'
+            }
+        });
+        let categorys = await this.fetchJSON(request);
+
+        let uri;
+        let pages;
+        let mangas = [];
+        for (const category of categorys.rows) {
+            if (category.more_btn.link.startsWith('/book/search/')) {
+                uri = new URL(category.more_btn.link, this.url);
+                pages = 1;
+                for (let page = 1; page <= pages; page++) {
+                    request = new Request(new URL('/api/v1/search/', this.url), {
+                        method: 'POST',
+                        body: '{"free":["1","2"],"page":'+page+',"sort_by":"0","tag":["'+uri.searchParams.get('tag')+'"]}',
+                        headers: {
+                            'Content-Type': 'application/json;charset=utf-8',
+                        }
+                    });
+
+                    let pageContent = await this.fetchJSON(request);
+                    pages = pageContent.max_page;
```

Comment:
I would expect that this is always the same, so it might be requested outside of the for-loop

Reply:
I'll always start with the assumption of one page and will read the actual max page from the response of page 1. I opted for this approach to only 1 query instead of 2 querys since I'll anyways receive the pages and mangas in the same JSON object.